code/modules/surgery/bodyparts/species_parts/misc_bodyparts.dm

```diff
@@ -558,3 +558,22 @@
 	unarmed_damage_low = 7
 	unarmed_damage_high = 21
 	unarmed_stun_threshold = 11
+
+///flesh
+
+/obj/item/bodypart/arm/left/flesh
```

Comment:
Would be funny to spawn a mob inside of these in case the limb gets added by a bioscrambler or something (and displaced by one attaching itself) but treat this as an optional niche suggestion.